### ISSUE TITLE
Can't get ReactPlayer to work with redirect urls

### ISSUE THREAD

**Issue by vincentchin** [CookPete/react-player#86]
I'm not sure how to get ReactPlayer to load a url like `https://domain.com/redirects/files/20`. 

Right now it returns a video tag in FilePlayer but no src is being set. Any ideas how to config this?

**Comment by CookPete** [comment]
This is something I've been thinking about recently, especially with the demand for supporting [blob URLs](https://github.com/CookPete/react-player/issues/77) and [RTSP URLs](https://github.com/CookPete/react-player/issues/83). I wonder if `ReactPlayer` should fall back to `FilePlayer` if a `url` is provided that no other player can play. Or perhaps a more explicit solution would be a `forcePlayer` prop to override the type of player to use, regardless of URL, like so:

```js
import ReactPlayer, { FilePlayer } from 'react-player'

return (
  <ReactPlayer
    url='https://domain.com/redirects/files/20'
    playing={true}
    forcePlayer={FilePlayer}
  />
)

**Comment by vincentchin** [comment]
Hmm, would it make more sense to just have `ReactPlayer` default to `FilePlayer` and not render any of the other player components unless their config is passed in? Otherwise, I think the idea you suggested is good.

**Comment by belohlavek** [comment]
Can we get _some_ implementation of this ASAP? Later on it'd possible to decide which method is more convenient!

Thanks making this component btw :grin:

**Comment by CookPete** [comment]
I've pushed a possible fix to the [`fileplayer-default`](https://github.com/CookPete/react-player/tree/fileplayer-default) branch, if travis build passes I'll merge it in and publish a new version. Probably should be a minor version update as we lose the static `canPlay` method (now that any URL can technically be "played").

**Comment by CookPete** [comment]
Fixed in `0.8.0`

**Comment by SmileHub** [comment]
Bringing up, ReactPlayer doesn't seem to follow redirects, for example:
[Livestream redirect](https://lb.ims-cdn.net/redirect/live/live_ims_27863_01?type=m3u8)
https://lb.ims-cdn.net/redirect/live/live_ims_27863_01?type=m3u8

tested with
`"react-player": "^2.13.0",`

also tested in demo page here:
[https://cookpete.com/react-player/](https://cookpete.com/react-player/)
